src/com/trainee/geetanjali/java/SimpleArraySum.java

```diff
@@ -4,23 +4,22 @@
 import java.util.Scanner;
 
 public class SimpleArraySum {
-    public static void main(String args[]) {
+    public static void main(String[] args) {
         int sum = 0;
         Scanner s = new Scanner(System.in);
-        System.out.println("Enter size of the array:");
+        System.out.print("Enter size of the array:");
         int n = s.nextInt();
```

Comment:
Catch the Exception.
If user inputs String instead on input here. It will throw Exception